src/main/java/Loopy.java

```diff
@@ -0,0 +1,191 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Loopy {
+    private static ArrayList<Task> tasks = new ArrayList<>(); // Create task list array
```

Comment:
comment is redundant, you can remove it

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#guideline-comment-minimally-but-sufficiently